src/main/java/racingcar/model/WinningCars.java

```diff
@@ -0,0 +1,36 @@
+package racingcar.model;
+
+import racingcar.constants.GameConstants;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class WinningCars {
+
+    private final List<Car> cars;
+
+    public WinningCars(List<Car> cars) {
+        this.cars = new ArrayList<>(cars);
+    }
+
+    public String getWinnerNames() {
+        return getWinnerList().stream()
+                .map(car -> " " + car.getName())
+                .collect(Collectors.joining(GameConstants.COMMA));
+    }
+
+    private List<Car> getWinnerList() {
+        return cars.stream()
+                .filter(car -> car.hasSameMoveCount(getWinnerMoveCount()))
+                .collect(Collectors.toList());
+    }
```

Comment:
현재 모든 cars를 순회할 때마다 매번 getWinnerMoveCount()가 호출될 것 같아요~!
```suggestion
    private List<Car> getWinnerList() {
        int winnerMoveCount = getWinnerMoveCount();

        return cars.stream()
                .filter(car -> car.hasSameMoveCount(winnerMoveCount))
                .collect(Collectors.toList());
    }
```
다음과 같이 수정하거나, 적절히 메서드를 분리해봐도 좋을 것 같네요~ 😀